src/platform/forms/save-in-progress/ApplicationStatus.jsx

```diff
@@ -139,20 +139,21 @@ export class ApplicationStatus extends React.Component {
 
         return (
           <div className="usa-alert usa-alert-info background-color-only sip-application-status vads-u-margin-bottom--2 vads-u-margin-top--0">
-            <h5 className="form-title saved">Your {appType} is in progress</h5>
+            <h5 className="form-title saved">
+              {`Your ${appType} is in progress`}
```

Comment:
Could this instead be `Your {appType} is in progress` without the extra set of brackets and backticks? Seems like this new formatting was applied everywhere you made a change, but I don't see that it's needed.

Reply:
As noted in our [Slack thread](https://dsva.slack.com/archives/CBU0KDSB1/p1678124939099199?thread_ts=1678121632.471959&cid=CBU0KDSB1), this `ApplicationStatus` component probably isn't used any more if at all... and also Josh was exploring the idea of wrapping content in a `role="text"` which I just tested in the `SaveInProgress` header and it still split the text into chunks... I think I'll just close this issue and we can come back to it once we have a better understanding of the issue and a definite fix.